lib/galaxy/authnz/psa_authnz.py

```diff
@@ -52,6 +53,7 @@
     "google": "google-openidconnect",
     "globus": "globus",
     "elixir": "elixir",
+    "nfdi": "helmholtz",
```

Comment:
Really not sure about that one, but no clue how to test.

Reply:
Maybe `"infraproxy"`? The [endpoint](https://github.com/python-social-auth/social-core/blob/345ad5ffd786ae11c430dfee2d8b1f58df37be49/social_core/backends/nfdi.py#L33C6-L33C16) matches the [well_known_oidc_config_uri](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1513/files#diff-3c0a42b6ddf2ec9362a8139c136892fe526bc1d771d90456fc7dd64db96f6ca7R77) you added.